src/ConfigEditor/Auth/tls/SelfSignedCertificate.tsx

```diff
@@ -1,10 +1,9 @@
-import React from "react";
-import { cx } from "@emotion/css";
-import { InlineField } from "@grafana/ui";
-import { InlineLabel } from "../../ConfigEditor";
-import { TLSSettingsSection } from "./TLSSettingsSection";
-import { SecretTextArea } from "../../../unreleasedComponents/SecretTextarea";
-import { useCommonStyles } from "../styles";
+import React from 'react';
+import { cx } from '@emotion/css';
+import { InlineField, SecretTextArea } from '@grafana/ui';
```

Comment:
Are you sure this change to use `SecretTextArea` from `@grafana/ui` is gonna work across the versions of Grafana that plugins need to support?

Reply:
In the previous PR I copy&pasted these components from `@grafana/ui` and mentioned that they are available starting from grafana 9.2. Since `@grafana/experimental` already relies on `@grafana/ui` version `^9.2.0`, I just removed those copies and used the ones from `@grafana/ui`.
Should be fine I think.